sdk/src/ssi/did.go

```diff
@@ -9,7 +9,8 @@ import (
 )
 
 type generateDIDKeyResult struct {
-	DIDKey            string  `json:"didKey"`
+	DID               string  `json:"did"`
```

Comment:
@andresuribe87 I know there was some confusion on my part on naming convention here. LMK if this is correct!

Reply:
This makes a lot of sense to me!